content/posts/0006-build-plan-buildpack.md

```diff
@@ -0,0 +1,93 @@
+---
+title: What is the Build Plan Buildpack and why should I care?
+date: "2021-02-09"
+slug: build-plan-buildpack-exploration
+author: feckhardt
+---
+The [Build Plan buildpack](https://github.com/paketo-community/build-plan)
+gives you very precise control over what buildpacks run during a build and what
+dependencies end up in the final image. Let's talk about how it works and
+why/when you would utilize this kind of functionality.
```

Comment:
Maybe add a short line about how it makes things more "customizable". I feel like it's more enticing for the intro, since that's the main thing users are looking for when they need the build plan buildpack

Reply:
What exactly are you thinking in terms of driving how the customization?

Reply:
Bump